src/components/Choice/src/ChoiceOption.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+	<button
+		:class="[
+			$s.Button,
+			{ [$s.selected]: isSelected },
+		]"
+		@click="selectSelf"
+	>
+		<slot />
+	</button>
+</template>
+
+<script>
+import key from './key';
+
+export default {
+	inject: {
+		controlState: key,
+	},
+
+	props: {
+		value: {
+			type: undefined,
+			required: true,
```

Comment:
How can a type that is undefined also be required?

Reply:
All of the docs examples show using `String`s as values, but @sohnj877 is your intent here to allow any possible value, e.g. `Number`s, `Object`s, `String`s, `Array`s?

Reply:
@pretzelhammer Correct